Fix/remove hardcoded npm package (#9410)
laravel-filament/filament: packages/panels/docs/11-themes.md

php artisan make:filament-theme admin
```

> By default, this command will use NPM to install dependencies. If you want to use a different package manager, you can use the `--pm` option:

Reviewer: ```suggestion
By default, this command will use NPM to install dependencies. If you want to use a different package manager, you can use the `--pm` option:

```

php artisan make:filament-theme admin
```

> By default, command will use node package manager with npm. If you want to use different package manager use `--pm` option:

Reviewer: ```suggestion
By default, this command will use NPM to install dependencies. If you want to use a different package manager, you can use the `--pm` option:
```
